org.ekstep.reviewercomments-1.0/editor/plugin.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Plugin to get comments from portal
+ * @class reviewerComments
+ * @constructor
+ * @extends org.ekstep.contenteditor.basePlugin
+ * @author Revathi P 
+ * @listens stage:render:complete
+ */
+
+org.ekstep.contenteditor.basePlugin.extend({
+    NO_COMMENTS: 'No Reviewer Comments',
+    initialize: function () {
+        ecEditor.addEventListener("stage:render:complete", this.initializeComments, this);
+    },
+    initializeComments: function (event, callback) {
+        var instance = this;
+        if (instance.comments == undefined) {
+            this.context = org.ekstep.contenteditor.globalContext;
+            if (!ecEditor._.isUndefined(this.context)) {
+                if (!ecEditor._.isUndefined(this.context.contentId) && this.context.contentId != "") {
+                    var instance = this;
```

Comment:
why are u declaring multiple instance variable in the same method?

Reply:
I need the `instance.comments` to be accessed globally, so i have declared `instance` here